Fix iverilog compile

See below;
```
$PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ -DHAVE_CONFIG_H -I. -Ilibmisc -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -Wall -Wextra -Wshadow   -Wno-deprecated-declarations -MD -c t-dll-analog.cc -o t-dll-analog.o
mv t-dll-proc.d dep/t-dll-proc.d
gcc -Wall -Wextra -Wshadow -Wstrict-prototypes  -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/iverilog-s20150603 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -o version.exe -I. -I. ./version.c
gcc: error: unrecognized command line option ‘-fstack-protector-strong’
gcc: error: unrecognized command line option ‘-fno-plt’
make: *** [version.exe] Error 1
make: *** Waiting for unfinished jobs....
mv t-dll-analog.d dep/t-dll-analog.d
Traceback (most recent call last):
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 456, in main
    execute(sys.argv[1:])
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 447, in execute
    verify=args.verify, variants=args.variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/api.py", line 208, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 2314, in build_tree
    notest=notest,
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 1477, in build
    cwd=src_dir, stats=build_stats)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/utils.py", line 374, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/utils.py", line 354, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/iverilog_1555610365274/work/conda_build.sh']' returned non-zero exit status 2.
The command "bash $TRAVIS_BUILD_DIR/.travis/script.sh" exited with 1.
```

--- a/iverilog/build.sh
+++ b/iverilog/build.sh
@@ -7,7 +7,7 @@ if [ x"$TRAVIS" = xtrue ]; then
 	CPU_COUNT=2
 fi
 
-export CXXFLAGS=-Wno-deprecated-declarations
+export CC_FOR_BUILD=$CC
 
 sh ./autoconf.sh
 ./configure --prefix=$PREFIX

